examples/application/app_with_build.py

```diff
@@ -7,10 +7,10 @@
 '''
 
 import kivy
-kivy.require('1.0.7')
 
 from kivy.app import App
 from kivy.uix.button import Button
+kivy.require('1.0.7')
```

Comment:
again better remove completely. which then makes `import kivy` redundant as most of the things are imported in this style:

    from kivy.x.y import Z